compute/admin_guide/vulnerability_management/detect_vulns_unpackaged_software.adoc

```diff
@@ -1,19 +1,34 @@
-== Detect vulnerabilities in unpackaged software
+== Vulnerabilities Detection
+
+=== Supported packages and languages
+
+Scan reports have a Package info tab, which lists all the packages installed in an image or host.
+The following list shows the package types that are currently supported and can be seen in scan results, by the name they are shown in the scan.
+
+* *Package* - supported Operating Systems packages, such as an RPM (Red Hat and derived distributions), dpkg/deb (Debian and derived distributions), or apk (Alpine Linux).
+* *Jar* - the Java Archive format, which is a zip file with a standard structure. The war file format, or web app archive, is also supported.
+* *Python* - a Python library.
+* *Nodejs* - a Node.js library.
+* *Gem* - a Ruby gem library.
+* *Go* - a GoLang library
+* *App* - a binary associated with a well-known application, such as Nginx or PostgreSQL. A full list of supported applications is listed below.
+
+Prisma Cloud uses a variety of approaches for package detection, these are purpose-built differently for images and hosts. For example, the host Defender only scans applications, as well as language-based packages, if the processes are running. The Windows Defender only scans packages that are installed with a package manager, missing Microsoft hotfixes, and .net framework applications.
+
+=== Unpackaged software
 
 Typically, the software is added to container images and hosts with a package manager, such as `apt`, `yum`, and `npm`.
 Prisma Cloud has a diverse set of upstream vulnerability data sources covering many different package managers across operating systems, including coverage for `Go`, `Java`, `Node.js`, `Python`, and `Ruby` components.
 Prisma Cloud typically uses the package manager's metadata to discover installed components and versions, comparing this data to the data in the Intelligence Stream's real-time CVE feed.
-The host defender only scans the running applications that are installed with a package manager on the host.
 
 Sometimes, you might install software without a package manager.
-For example, the software might be built from source and then added to an image with the Dockerfile `ADD` instruction, or your developers might unzip software from a tarball to a location on a host, and utilize the application.
+For example, the software might be built from a source and then added to an image with the Dockerfile `ADD` instruction, or your developers might unzip software from a tarball to a location on a host, and utilize the application.
 In these cases, there is no package manager data associated with the application.
 
 Prisma Cloud uses a variety of analysis techniques to detect metadata about software not installed by package managers. These are purpose-built differently for images and hosts.
```

Comment:
We have the same sentence above, let's remove the "These are purpose-built differently for images and hosts."

Reply:
@grevach Can you please review as well?